proto-x86/helpers.sh

```diff
@@ -9,20 +9,20 @@ function qx86stop() {
 }
 export -f qx86stop
 
-alias qx86cli='docker exec qx86 qcli'
+alias qx86cli='docker  exec qx86 qtum-cli -regtest'
 
 function qx86deploy() {
-    docker exec -t qx86 deploy_contract `cat $1`
+    docker exec -t qx86 qtum-cli -regtest createcontract `cat $1`
 }
 export -f qx86deploy
 
-function qx86tb() {
-    docker run --rm -v "${PWD}:/root/bind" qtumx86 x86tb
-}
-export -f qx86tb
-
 function qx86make() {
-    docker run --rm -v "${PWD}:/root/bind" qtumx86 qmake "$@"
+    docker run --rm -v "${PWD}:/root/bind" -w "/root/bind" qtumtoolchain-alpine make
```

Comment:
The ""$@"" is needed so that `qx86make clean` for example works

Reply:
@VoR0220 Hi, once this is resolved, I am able to merge the PR.